Edits to PR5751: TDE on BigAnimal with Customer Managed Keys (#5817)
EnterpriseDB/docs: product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

Enable **Transparent Data Encryption (TDE)** to use your own encryption key. This option is available for EDB Postgres Advanced Server and EDB Postgres Extended Server for version 15 and later. Select an encryption key from your project and region to encrypt the cluster with TDE. To learn more about TDE support, see [Transparent Data Encryption](../../overview/03_security/#your-own-encryption-key---transparent-data-encryption-tde).

!!!Note "Important"
- To enable and use TDE for a cluster, you must first enable the encryption key and add it at the project level before creating a cluster. 
  To add a key, see [Adding a TDE key at project level](../../administering_cluster/projects.mdx/#adding-a-tde-key).
- To enable and use TDE for a cluster, you must complete the configuration on the platform of your key management provider after creating a cluster. See [Completing the TDE configuration](#completing-the-TDE-configuration) for more information.
!!!

#### Completing the TDE configuration

> After you create the cluster in the BigAnimal console, the cluster is in the **Waiting for access to encryption key** state. To complete the configuration and enable the key sync between BigAnimal and the key management platform, you must grant encrypt and decrypt permissions to your key:

Author: Is this edit okay? I'd rather we stay away from using the UI as the subject.

Reviewer: Sounds good! I think with the rebranding into EDB Postgres AI, we could start calling it Console or EDB Postgres AI Console whenever necessary. Here, I like the edit with "cluster".

After you create the cluster in the BigAnimal console, the UI will display the **Waiting for access to encryption key** state. To complete the configuration and enable the key sync between BigAnimal and the key management platform you must grant encrypt and decrypt permissions to your key:
After you create the cluster in the BigAnimal console, the cluster is in the **Waiting for access to encryption key** state. To complete the configuration and enable the key sync between BigAnimal and the key management platform, you must grant encrypt and decrypt permissions to your key:

1.  In BigAnimal, select the cluster name and access the cluster's page. See the **Action required: grant key permissions to activate the cluster**.

> 1.  Copy the **Principal** identifier (AWS), **service account** (GCP) or **MSI Workload Identity** (Azure) to your clipboard. 

Author: I deleted this because the step is repeated in the details that follow.

Reviewer: Good catch.

> 1.  Follow the on-screen guide to grant encrypt and decrypt permissions to your key. Some additional guidance:

Reviewer: ```suggestion
1.  Follow the on-screen guide to grant encrypt and decrypt permissions to your key. The instructions differ depending on the cloud provider of your key. Some additional guidance:
```

    1.  Copy the **Principal** identifier to your clipboard. 
    1.  Go to the AWS console, and navigate to the **Key Management Service**.
    1.  Select **Customer-managed keys**, and **Edit policy** for your key.

>     1.  Append a new policy statement where the `Principal.AWS` field contains the **Principal** identifier you copied to your clipboard and the `Principal.Action` field contains **kms:Encrypt** and **kms:Decrypt** permissions. 

Author: ```suggestion
    1.  Append a new policy statement where the `Principal.AWS` field contains the **Principal** identifier you copied earlier and the `Principal.Action` field contains **kms:Encrypt** and **kms:Decrypt** permissions. 
```

    <details><summary>GCP</summary>

    1.  Copy the **service account** to your clipboard.

>     1.  On the Google Cloud console, select **Security**, **VIEW BY PRINCIPALS**, and **GRANT ACCESS** for your key.

Author: I wasn't sure of this edit. It was a bit difficult to follow this step. Are these three options to select? It's unclear to me as a reader what this meant.

Reviewer: Yes, it is three options depending on the cloud provider for your key. Let me add a suggestion to make this clearer at the top of this step.
